e2e/interchaintestv8/relayer_test.go

```diff
@@ -856,4 +870,20 @@ func (s *RelayerTestSuite) ICS20TimeoutFromCosmosTimeoutTest(
 			s.Require().Equal(testvalues.InitialBalance, resp.Balance.Amount.Int64())
 		}))
 	}))
+
+	s.Require().True(s.Run("Verify no balance on Ethereum", func() {
+		denomOnEthereum := transfertypes.NewDenom(transferCoin.Denom, transfertypes.NewHop(transfertypes.PortID, testvalues.CustomClientID))
+
+		_, err := s.ics20Contract.IbcERC20Contract(nil, denomOnEthereum.Path())
+		// Ethereum side did not received the packet, ERC20 contract corresponding to the denom does not exist
+		s.Require().Error(err)
+		s.Require().Contains(err.Error(), "execution reverted: custom error 0xe1275e2f")
+	}))
+
+	s.Require().True(s.Run("Receive packets on Ethereum after timeout should fail", func() {
+		ics26Address := ethcommon.HexToAddress(s.contractAddresses.Ics26Router)
+		receipt, err := eth.BroadcastTx(ctx, s.EthRelayerSubmitter, 5_000_000, ics26Address, txBodyBz)
+		s.Require().NoError(err)
+		s.Require().NotNil(receipt)
```

Comment:
This doesn't look like it is failing? Is this correct?

Reply:
I think this should fail, not sure why this is returning no error.  Maybe the contract gracefully finishes the execution instead of reverting the transaction.

Reply:
I think this is not failing only in the quick CI where we use mock light clients. Can we run this test case only in POS mode?